Facebook.iOS/AudienceNetwork/source/Facebook.AudienceNetwork/ApiDefinition.cs

```diff
@@ -86,6 +86,237 @@ interface AdChoicesView {
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordPets;
+		[Field ("FBAdExtraHintKeywordPets", "__Internal")]
+		NSString Pets { get; }
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordPhotography;
+		[Field ("FBAdExtraHintKeywordPhotography", "__Internal")]
+		NSString Photography { get; }
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordPolitics;
+		[Field ("FBAdExtraHintKeywordPolitics", "__Internal")]
+		NSString Politics { get; }
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordRealEstate;
+		[Field ("FBAdExtraHintKeywordRealEstate", "__Internal")]
+		NSString RealEstate { get; }
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordRoleplayingGames;
+		[Field ("FBAdExtraHintKeywordRoleplayingGames", "__Internal")]
+		NSString RoleplayingGames { get; }
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordScience;
+		[Field ("FBAdExtraHintKeywordScience", "__Internal")]
+		NSString Science { get; }
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordShopping;
+		[Field ("FBAdExtraHintKeywordShopping", "__Internal")]
+		NSString Shopping { get; }
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordSociety;
+		[Field ("FBAdExtraHintKeywordSociety", "__Internal")]
+		NSString Society { get; }
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordSports;
+		[Field ("FBAdExtraHintKeywordSports", "__Internal")]
+		NSString Sports { get; }
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordTechnology;
+		[Field ("FBAdExtraHintKeywordTechnology", "__Internal")]
+		NSString Technology { get; }
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordTelevision;
+		[Field ("FBAdExtraHintKeywordTelevision", "__Internal")]
+		NSString Television { get; }
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordTravel;
+		[Field ("FBAdExtraHintKeywordTravel", "__Internal")]
+		NSString Travel { get; }
+
+		// extern const FBAdExtraHintKeyword _Nonnull FBAdExtraHintKeywordVideoComputerGames;
+		[Field ("FBAdExtraHintKeywordVideoComputerGames", "__Internal")]
+		NSString VideoComputerGames { get; }
+	}
+
+	// @interface FBAdExtraHint : NSObject
+	[BaseType (typeof (NSObject), Name = "FBAdExtraHint")]
+	interface AdExtraHint {
+		// @property (copy, nonatomic) NSString * _Nullable UISpec;
+		[NullAllowed, Export ("UISpec")]
+		string UISpec { get; set; }
+
+		// @property (copy, nonatomic) NSString * _Nullable contentURL;
+		[NullAllowed, Export ("contentURL")]
+		string ContentUrl { get; set; }
+
+		// @property (copy, nonatomic) NSString * _Nullable extraData;
+		[NullAllowed, Export ("extraData")]
+		string ExtraData { get; set; }
+
+		// -(instancetype _Nonnull)initWithKeywords:(NSArray<FBAdExtraHintKeyword> * _Nonnull)keywords;
+		[Export ("initWithKeywords:")]
+		IntPtr Constructor (string [] keywords);
```

Comment:
This is a `NSArray<FBAdExtraHintKeyword>` so it needs keys from `AdExtraHintKeywords`

```suggestion
		IntPtr Constructor (NSString [] keywords);
```

Reply:
You have a point, good sir! Changing it using beta tool.